presto-main/src/main/java/com/facebook/presto/sql/planner/RelationPlanner.java

```diff
@@ -538,7 +551,8 @@ private RelationPlan planLateralJoin(Join join, RelationPlan leftPlan, Lateral l
         PlanBuilder leftPlanBuilder = initializePlanBuilder(leftPlan);
         PlanBuilder rightPlanBuilder = initializePlanBuilder(rightPlan);
 
-        PlanBuilder planBuilder = subqueryPlanner.appendLateralJoin(leftPlanBuilder, rightPlanBuilder, lateral.getQuery(), true, LateralJoinNode.Type.INNER);
+        PlanBuilder planBuilder =
```

Comment:
nit: usually we put everything in one line or follow this pattern:
```java
PlanBuilder planBuilder =  subqueryPlanner.appendLateralJoin(
    leftPlanBuilder,
    rightPlanBuilder,
    subQueryNotSupportedError(lateral.getQuery()),
    true,
    LateralJoinNode.Type.INNER);
```
Just like what you have done to `appendApplyNode`